docs/source/api.rst

```diff
@@ -0,0 +1,60 @@
+*****************
+API Documentation
+*****************
+
+.. currentmodule:: merlin.systems
+
+
+Workflow Graph Constructors
+---------------------------
+
+.. currentmodule:: merlin.systems.dag
+
+.. autosummary::
+   :toctree: generated
+
+   Ensemble
+   Node
+
+Workflow Operator Constructors
+------------------------------
+
+.. currentmodule:: merlin.systems.dag.ops
+
+.. autosummary::
+   :toctree: generated
+
+   workflow.TransformWorkflow
+   operator.InferenceDataFrame
+   tensorflow.PredictTensorflow
+   softmax_sampling.SoftmaxSampling
+   session_filter.FilterCandidates
+   unroll_features.UnrollFeatures
+
+.. faiss.QueryFaiss
+.. feast.QueryFeast
+
+
+Workflow Functions
```

Comment:
From here down, I don't think we expect users to interact with any of these directly. (We should probably make them private.)